Themis/Views/Assessment/CodeEditor/CodeView.swift

```diff
@@ -37,9 +37,7 @@ struct CodeView: UIViewControllerRepresentable {
         func textViewDidChangeSelection(_ textView: TextView) {
```

Comment:
Needs to be wrapped with @MainActor and run on the main thread as its publishing changes to the view